EZP-27538 delete translations

mcurran-inviqa opened the pull request.

maxbaldanza commented on src/bundle/Resources/views/content/tabs/translations.html.twig:
```diff
@@ -15,10 +19,12 @@
     </tbody>
 </table>
+
+{{ form_end(actionsForm) }}
```
Can we add a new line at the end please?

maxbaldanza commented on src/lib/Repository/Values/Content/UiLanguage.php:
```diff
@@ -0,0 +1,41 @@
+namespace EzSystems\HybridPlatformUi\Repository\Values\Content;
+
+use eZ\Publish\API\Repository\Values\Content\Language;
+
```
Can we add a doc block and `@property-read` for the new variables?

maxbaldanza commented on src/lib/View/Content/Translations/TranslationParameterSupplier.php:
```diff
@@ -0,0 +1,37 @@
+use eZ\Publish\Core\MVC\Symfony\View\ContentView;
+use EzSystems\HybridPlatformUi\Repository\UiTranslationService;
+use EzSystems\HybridPlatformUi\View\Content\ParameterSupplier;
+
```
Can we add a doc block?

maxbaldanza commented on src/lib/View/Content/Translations/TranslationParameterSupplier.php:
```diff
@@ -0,0 +1,37 @@
+<?php
```
No Spec?

maxbaldanza reviewed (changes requested):
Looks good, just a few doc block issues

maxbaldanza commented on src/bundle/Resources/views/content/tabs/translations.html.twig:
```diff
@@ -15,10 +19,12 @@
         {% for translation in translations %}
         <tr>
-            <td></td>
+            <td>{{ form_widget(actionsForm.removeTranslations[translation.languageCode], {'attr': {'disabled': not translation.canDelete}}) }}</td>
```
It looks like this is only checked on the front end. Do we not need some server side validation for this?

mcurran-inviqa replied:
the API blocks deletion of the main location and checks user permissions, no need to duplicate before making the call

maxbaldanza commented on src/lib/Mapper/Form/TranslationMapper.php:
```diff
@@ -0,0 +1,26 @@
+<?php
```
Can we add license and doc block to this?

maxbaldanza reviewed (changes requested):
Looks good apart from this small change 👍

mcurran-inviqa committed:
EZP-27538 delete translations
- Adds translation action form
- Moves translation tab view into TranslationController
- Handles deletion of translations via action form
